packages/table/src/table-column/render-helper.ts

```diff
@@ -127,7 +128,7 @@ function useRender(props: TableColumnCtx, slots, owner: ComputedRef<any>) {
           }
         }
         checkSubColumn(children)
-        return h('div', props, [prefix, children])
+        return h('div', props, [prefix, cloneDeep(children)])
```

Comment:
Will we have overhead when the children is large enough?

Reply:
The data rendered here is the data of the cell. This data is not very large. This is mainly aimed at solving the problem caused by memory address reference.